src/main/java/fi/helsinki/cs/tmc/cli/backend/Settings.java

```diff
@@ -49,32 +51,40 @@ public void setWorkDir(WorkDir workDir) {
 
     @Override
     public String getServerAddress() {
-        return account.getServerAddress();
+        if (account.getServerAddress().isPresent()) {
+            return account.getServerAddress().get();
+        }
+        return "https://tmc.mooc.fi";
```

Comment:
Maybe we should extract this into a constant?

Reply:
:tulip: